cmd/buildah/pull.go

```diff
@@ -57,6 +59,7 @@ func init() {
 	flags.StringVar(&opts.blobCache, "blob-cache", "", "store copies of pulled image blobs in the specified directory")
 	flags.StringVar(&opts.certDir, "cert-dir", "", "use certificates at the specified path to access the registry")
 	flags.StringVar(&opts.creds, "creds", "", "use `[username[:password]]` for accessing the registry")
+	flags.StringVar(&opts.pullPolicy, "policy", "missing", "missing, always, or never.")
```

Comment:
Naming for the option seems inconsistent with `build-using-dockerfile`.

Reply:
`podman pull --pull `
Feels like bad CLI, which is why I did not go with it.